tests/devapps/DesktopTestApp/MainForm.cs

```diff
@@ -362,6 +362,13 @@ private UIBehavior GetUIBehavior()
             return behavior;
         }
 
+        private bool GetForceRefresh()
+        {
+            bool forceRefresh = forceRefreshCheckBox.Checked;
```

Comment:
return forceRefreshCheckBox.Checked

Also, suggestion:  make this a property:  private bool IsForceRefreshEnabled => forceRefreshCheckBox.Checked;

Reply:
great feedback, thanks!